arcade/gui/widgets/slider.py

```diff
@@ -226,7 +226,7 @@ def do_render(self, surface: Surface):
         )
 
     def _cursor_pos(self) -> Tuple[int, int]:
-        return self.value_x, self.y + self.height // 2
+        return self.value_x, int(self.y + self.height // 2)
```

Comment:
value_x looks like a float, so instead of making self.y + self.height // 2 an int, change -> Tuple[int, int]: to -> Tuple[float, float]: